pkg/interface/chat/src/js/api.js

```diff
@@ -181,6 +181,31 @@ class UrbitApi {
     });
   }
 
+  metadataAction(data) {
+    console.log(data)
+    return this.action("metadata-store", "metadata-action", data);
+  }
+
+  metadataAdd(appPath, groupPath, title, description, dateCreated, color) {
+    let creator = `~${window.ship}`
+    return this.metadataAction({
+      add: {
+        "group-path": groupPath,
+        resource: {
+          "app-path": appPath,
+          "app-name": "chat"
+        },
+        metadata: {
+          title: title,
+          description: description,
+          color: color,
```

Comment:
Did you know that you can just write ‘color,’ instead of ‘color: color,’?

Reply:
Wtf? I mean I guess I get it because passing a single argument to the object wouldn’t have any other key but it assumes the argument is the key name?

Reply:
yeah, it makes the variable name the key